lib/sirius_api/strategies/local_token.rb

```diff
@@ -15,11 +15,14 @@ def store?
 
       def authenticate!
         if access_token.blank?
-          fail 'Missing access token'
-          return
+          return 'Missing access token'
```

Comment:
Raising exceptions in bang method is one of many bang method conventions, which is used in some projects and completely ignored in many others.

Also raising any exception in Warden authentication strategies will cause the whole authentication cascade to fail, which is excactly what you want to avoid right here.

Trust me, I've spent quite a time debugging this and scratching my head on why the hell it does not work properly.

The return error messages could be handled better though. I think I saw that Warden has some mechanism how to pass partial failure messages, just have to look more into that.